docs/dev/table/sql.md

```diff
@@ -1873,6 +1873,18 @@ TO_BASE64(string)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+LTRIM(string)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the result string which trimmed the left spaces from the base string; returns NULL if <i>string</i> is NULL.</p> 
+        <p>E.g., <code>LTRIM(" This is a test String.")</code> returns "This is a test String.".</p>
```

Comment:
\<code>LTRIM(" This is a test String.")\</code> => \<code>LTRIM(' This is a test String.')\</code>